tests/integration/__init__.py

```diff
@@ -111,6 +111,8 @@ def _get_dse_version_from_cass(cass_version):
 KEEP_TEST_CLUSTER = bool(os.getenv('KEEP_TEST_CLUSTER', False))
 SIMULACRON_JAR = os.getenv('SIMULACRON_JAR', None)
 
+default_cassandra_version = '3.11'
```

Comment:
This duplicates [code a couple lines down](https://github.com/datastax/python-driver/pull/933/files#diff-437a31ded249825f4640cfd1c0608bf8R119) -- is this intentional?

Reply:
Nope, good catch